apps/syntrest/src/main/java/no/nav/registre/syntrest/kubernetes/KubernetesController.java

```diff
@@ -53,6 +53,8 @@
     private RestTemplate noAuthRestTemplate;
     private RestTemplate authRestTemplate;
 
+    private final List<String> appsFlyttetTilGithub = Arrays.asList("synthdata-sam", "synthdata-frikort");
```

Comment:
Hvorfor ikke bare legge inn disse som "synt-sam" og "synt-frikort"?

Reply:
Kan også være en idé å etterhvert gjøre en github-query i stedet for å ha disse hardkodede verdiene. (Hardkoding = bad)

Reply:
Prøvde først å endre til synt-sam og synt-frikort, men det ble utrolig mange endringer som måtte gjøres siden syntrest er basert på appname på formatet "synthdata-app". Til slutt vil listen bli fjernet når alle synt-appene er flyttet over til testnorge-syntetiseringspakker og alle appene kan hentes fra samme sted.

Reply:
Ja, skjønner. De blir hetende synthdata-[...] i kube. Kan evt. legge inn problemstillingen som et issue på dette repoet, sånn at vi ikke glemmer å gjøre det når de andre er på plass.

Reply:
Godt poeng, jeg kan legge det til i trello i gjøremål på kortet "Opprette monorepo for alle syntpakker" ;)